ethcore/builtin/src/lib.rs

```diff
@@ -194,7 +195,13 @@ impl Pricer for ModexpPricer {
 
 		let adjusted_exp_len = Self::adjusted_exp_len(exp_len, exp_low);
 
-		let (gas, overflow) = Self::mult_complexity(m).overflowing_mul(max(adjusted_exp_len, 1));
+		let complexity_formula = if self.new_formula {
+			Self::mult_complexity_new
+		} else {
+			Self::mult_complexity
+		};
+
+		let (gas, overflow) = (complexity_formula)(m).overflowing_mul(max(adjusted_exp_len, 1));
 		if overflow {
 			return U256::max_value();
 		}
```

Comment:
(gas / self.divisor as u64).into() --> min((gas / self.divisor as u64).into(),100)

This sets a minimum gas price of 100

Reply:
FYI this is a couple rows down but it wouldn't let me comment on it for some reason

Reply:
EIP has been updated to set the minimum gas price at 200